src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/EventTypeAT.java

```diff
@@ -0,0 +1,170 @@
+package de.zalando.aruha.nakadi.webservice;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.zalando.aruha.nakadi.config.NakadiConfig;
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.EventTypeSchema;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Test;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.http.ContentType.JSON;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+
+public class EventTypeAT extends BaseAT {
+
+    static private final String ENDPOINT = "/event_types";
+    private final ObjectMapper mapper = (new NakadiConfig()).jacksonObjectMapper();
+
+    @Test
+    public void whenGETThenListsEventTypes() throws JsonProcessingException {
```

Comment:
The only problem I have here is that you use the nakadi application functionality to create data for the test. 

What I mean is:
Let's say you have a system that can do `get(X)` and `set(X)`.
I believe you can't use `set(X)` for bootstraping data in a test for `get(X)` (and vise versa), because if you will have a bug in both set and get - you have a change of not noticing it.
But in general that is also a tricky point, and we didn't discuss it within Nakadi project yet :)